docs/postman/vmpooler generic.postman_environment.json

```diff
@@ -0,0 +1,63 @@
+{
+	"id": "8c770a8b-db0d-4722-82d0-ec06712b9a44",
+	"name": "vmpooler generic",
+	"values": [
+		{
+			"key": "url",
+			"value": "https://myserver.com/",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "token",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "api_prefix",
+			"value": "api/v2",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "ldap_user",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "ldap_pass",
+			"value": "",
+			"type": "secret",
+			"enabled": true
+		},
+		{
+			"key": "ondemand_pool_name",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "pooled_pool_name",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "additional_disk_size",
+			"value": "8",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "pool_management",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		}
+	],
+	"_postman_variable_scope": "environment",
+	"_postman_exported_at": "2022-04-01T17:17:09.868Z",
+	"_postman_exported_using": "Postman/9.15.2"
+}
```

Comment:
Looks like you need a new line at the end but otherwise good

Reply:
this is an exported file straight from postman